File: desafio-02/TeijiW/javascript/desafio02.js
for(let n=1;n<=10000;n++){
    count = 0
    for (let i=1;i<=n;i++){
       if (n%i === 0){
         count++
       } 
    }
    if (count === 2){
      console.log(n)
    }
  }

File: desafio-02/Harrisoned/python/desafio02.py
"""
Author: Lucas
Purpose: Answer to challenge 02
"""
NUMEROS = range(10000)
for numero in NUMEROS:
    div = 1
    count = 0
    while div <= numero:
        if numero%div == 0:
            count += 1
        if div >= numero:
            break
        div += 1
    if count >= 2:
        if count >= 3:
            pass
        else:
            print(numero)
